docs/ui/insight/tools.md

```diff
@@ -0,0 +1,109 @@
+
+  <img src="../images/Workbench_Graph.png">
+
+* Visualizations of your [RedisTimeSeries](https://oss.redis.com/redistimeseries/) data.
+
+  <img src="../images/Workbench_TimeSeries.png">
+
+## Database analysis
+
+Use the database analysis tool to optimize the performance and memory usage of your Redis database. Check data type distribution and memory allocation and review the summary of key expiration time and memory to be freed over time. Inspect the top keys and namespaces sorted by consumed memory or key length and count of keys, respectively. Capture and track the changes in your database by viewing historical analysis reports. Next figure shows a sample database analysis report.
+
+<img src="../images/database_analysis.png">
+
+## Redis Streams support
+
+Create and manage streams by adding, removing, and filtering entries per timestamp. To see and work with new entries, enable and customize the automatic refresh rate.
+
+View and manage the list of consumer groups. See existing consumers in a given consumer name as well as the last messages delivered to them. Inspect the list of pending messages, explicitly acknowledge the processed items, or claim unprocessed messages via RedisInsight.
+
+<img src="../images/streams.png">
+
+## Search capabilities
+
+If you're using Redis Stack's indexing, querying, or full-text search capabilities, RedisInsight provides UI controls to run search queries against a preselected index. You can also create a secondary index of your data in a dedicated pane.
+
+<img src="../images/search.png">
+
+## Bulk actions
+
+Easily and quickly delete multiple keys of the same type and/or with the same key name pattern in bulk. To do so, in the List or Tree view, set filters per key type or key names and open the Bulk Actions section. The section displays a summary of all the keys with the expected number of keys that will be deleted based on the set filters.
+
+When the bulk deletion is completed, RedisInsight displays the results of this operation with the number of keys processed and the time taken to delete the keys in bulk.
+Use bulk deletion to optimize the usage of your database based on the results from the Redis database analysis.
+
+<img src="../images/bulk_actions.png">
+
+## Slow Log
+
+The Slow Log tool displays the list of logs captured by the `SLOWLOG` command to analyze all commands that exceed a specified runtime, which helps with troubleshooting performance issues. Specify both the runtime and the maximum length of Slowlog (which are server configurations) to configure the list of commands logged and set the auto-refresh interval to automatically update the list of commands displayed.
+
+<img src="../images/slowlog.png">
+
+## Plugins
+
+With RedisInsight you can now also extend the core functionality by building your own data visualizations. See our [plugin documentation](https://github.com/RedisInsight/RedisInsight/wiki/Plugin-Documentation) for more information.
```

Comment:
we can call them Create your own custom data visualizations

plugins term is a bit vague